n2y/blocks.py

```diff
@@ -117,8 +117,6 @@ def to_pandoc(self):
                 properties = self.page.properties_to_values()
             else:
                 properties = {}
-            if 'title' not in properties:
```

Comment:
Nice, yeah I don't know why we were adding the title like this ...

Reply:
Do we mention the "title" property in the readme? If so, we should update it.

Reply:
Nope, not this property, all clear